pkg/storage/ingest/fetcher.go

```diff
@@ -66,22 +66,27 @@ type fetchWant struct {
 	startOffset    int64 // inclusive
 	endOffset      int64 // exclusive
 	bytesPerRecord int
```

Comment:
[nit] You know I'm obsessed about calling the same data with the same exact name across the code based :D 
```suggestion
	estimatedBytesPerRecord int
```